core/src/main/java/org/radargun/reporting/ReporterHelper.java

```diff
@@ -83,7 +85,8 @@ public static void loadReporters(File reporterDir, Map<String, Class<? extends R
                } else {
                   log.debug("Registering reporter '" + type + "' using " + clazzName);
                   try {
-                     Class<? extends Reporter> reporterClazz = (Class<? extends Reporter>) classLoader.loadClass(clazzName);
+                     ClassLoader reporterHelperClassLoader = new URLClassLoader(urls.toArray(new URL[urls.size()]), ReporterHelper.class.getClassLoader());
```

Comment:
Why do you create another ClassLoader and don't use the reporterSpecificClassloader?